infrastructure/iac/template/index.html

```diff
@@ -82,7 +83,17 @@ <h2>References</h2>
       {{end}}
     </div>
   </section>
+  <!-- Ignore In Line Actions -->
+  {{if not .Issue.IsIgnored}}
+  <footer id="ignore-container" class="ignore-action-container hidden">
+    <div id="ignore-actions">
+      <div class="actions row">
+        <button id="ignore-file-issue" issue="{{.Issue.ID}}" path="{{.ResourcePath}}" filePath="{{.FilePath}}" class="ignore-button secondary">Ignore in this file</button>
+      </div>
+    </div>
+  </footer>
+  {{end}}
```

Comment:
question: what is `end`ed here?

Reply:
That is the end of the {{if not .Issue.IsIgnored}} statement.